pulsar-client/src/main/java/org/apache/pulsar/client/impl/AbstractBatchMessageContainer.java

```diff
@@ -76,6 +79,11 @@ public long getCurrentBatchSize() {
         return currentBatchSizeBytes;
     }
 
+    @VisibleForTesting
+    public int getMaxBatchSize() {
```

Comment:
```suggestion
   int getMaxBatchSize() {
```

Testing method is better to have the narrowest visibility :)